build.xml

```diff
@@ -159,6 +159,16 @@
     </target>
 
     <!-- Public API -->
+    <target name="spring4-soffit" description="Rebuild uPortal-soffit packages with Spring 4">
```

Comment:
Using one build system to modify another build system seems risky to me.
Could this be managed completely within `gradle`?